DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/AccessibilityCollection.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace DigitalLearningSolutions.Web.AutomatedUiTests.AccessibilityTests
+{
+    using Xunit;
+
+    [CollectionDefinition("Accessibility collection")]

```

Comment:
Could we name this "Selenium test collection" as we probably don't want to restart the server for other tests (if we extend things and use these tests for stuff other than accessibility)

Reply:
Super minor, but should we change the class name too?